docs/self-signed-certificate.md

```diff
@@ -8,7 +8,7 @@ To keep consistency with velero, Velero plugin for vsphere also supports using v
 * Guest cluster: supported since v1.1.1
 * Supervisor cluster: not supported
 
-To install with a storage provider secured by a self-signed certificate, the --cacert option needs to be added to provide a path to a certificate bundle to trust.
+To install with a storage provider secured by a self-signed certificate, the --cacert option needs to be added to provide a path to a certificate bundle to trust. This certificate is used to verify TLS connection to the storage provider when backing up and restoring.
```

Comment:
Please clarify that this is for the Backup Storage Location such as an object store.

Reply:
Can you also add a note at the top doc level where this is referenced?

Reply:
@xing-yang  This doc is referenced by guest.md, supervisor.md and vanilla.md. What should we put in the note?